app/policies/playlist_activity_policy.rb

```diff
@@ -0,0 +1,3 @@
+class PlaylistActivityPolicy < StudentManagementPolicy
+  alias_method :new?, :index?
```

Comment:
put this in the `StudentManagementPolicy` interface if it is the same in both cases.